LICENSE.md

```diff
@@ -0,0 +1,410 @@
+# GeoServer License Notice
+
+GeoServer is distributed under the GNU General Public License Version 2.0 license:
+
+    GeoServer, open geospatial information server
+    Copyright (C) 2014-2020 Open Source Geospatial Foundation.
```

Comment:
```suggestion
    Copyright (C) 2014-2023 Open Source Geospatial Foundation.
```

Or perhaps no endyear

Reply:
I can process these files to include current year like I do for the header and footer.